bundle-size/test/api.test.js

```diff
@@ -673,13 +681,19 @@ describe('bundle-size api', () => {
         )
         .reply(418, 'I am a tea pot');
 
+      // Silence expected error messages during testing.
+      const logLevel = probot.logger.level();
```

Comment:
*Here* I would prefer that you mock it instead of changing the log level (so that if something changes in the code and the test fails, we can still get some idea as to why - i.e. by expecting the log to be have been called)

Reply:
Done.